requirements.txt

```diff
@@ -1,6 +1,6 @@
 qiskit>=1.0,<2.0
 numpy>=2.0
-scipy>=1.4,<1.16
+scipy>=1.16
```

Comment:
This cannot stay like this, > 1.4 I think was ok as in Python 3.9 1.16 does not exist. If you want to bump the min to say the prior version that still works with 3.9 that should be fine.